phing/common.xml

```diff
@@ -103,24 +112,70 @@ Akeeba Build Files
 
 	<!--
 	Set up build templates by copying /build/templates and replacing tokens
+
+	Build templates for Core and Pro versions are supported. In order to use this you only have to
+	add 'core' and 'pro' suffixes to the manifest files. The Core and Pro versions should have the
+	same filename, followed by the version suffix ('-core' or '-pro'). The file will be copied and
+	the suffix will be removed automatically. For example:
+
+	/build/templates/component/something-core.xml -> /component/something.xml
+	/build/templates/component/something-pro.xml  -> /component/something.xml
+
+	The setup-templates-corepro task should be called every time before the extension is zipped,
+	in order to make sure the correct xml file is used.
+	-->
+
+	<!--
+	Setup general build templates which apply always
 	-->
-	<target name="setup-templates" description="Setup build templates" depends="setup-properties">
+	<target name="setup-templates-general" description="Setup general build templates"
+					depends="setup-properties">
 		<copy todir="${dirs.root}" overwrite="true">
 			<fileset dir="${dirs.build.templates}">
 				<include name="**" />
+				<exclude name="**_core.xml" />
+				<exclude name="**_pro.xml" />
 				<exclude name="link.php" />
```

Comment:
The test was good but I've overlooked this and I think I also have to change this to **/*_core.xml in stead of **_core.xml. Same for the pro.